### PR TITLE
address minor nitpicks and add Micro-ROS as a possible extension.

### PR DESCRIPTION
**Pull request by fujitatomoya**

### CODE REVIEW

File: sig-robotics/charter.md [hunk 1]
```diff
@@ -6,27 +6,28 @@ This charter adheres to the conventions described in [KubeEdge Open Governance](
 
 The SIG Robotics focuses on technical discussion, API definition, reference architecture, implementation in robots, to empower existing robotics ecosystems with KubeEdge, based on the edge-cloud synergy architecture, to improve robot intelligence and development efficiency.
 
-### In scope
```
Comment (fujitatomoya) [inline sig-robotics/charter.md:9]: sounds redundant so that removed.

File: sig-robotics/charter.md [hunk 2]
```diff
@@ -6,27 +6,28 @@ This charter adheres to the conventions described in [KubeEdge Open Governance](
...
+  - Cooperate with other open source communities, e.g.,  [ROS](https://www.ros.org/), [Gazebo](http://gazebosim.org/), also with other SIGs like [SIG AI](https://github.com/kubeedge/community/tree/master/sig-ai) and [SIG IoT](https://github.com/kubeedge/community/tree/master/sig-device-iot)
+- Synergy mechanisms research for robotics workloads, **including but not limited to**:
+  - Cloud environment for robot programming, compilation, simulation, packaging and deployment
+  - Integration with cloud services like AI training, data storage, monitoring
+  - Edge-cloud synergy system for robots
+    - Robot application deployment to robot base on edge
+    - Fleet management and control
+    - Teleoperation / Remote control
+    - Cloud service integration like AI(training and inference), data storage, monitoring
+    - Additional protocol extension like [Micro-XRCE-DDS](https://github.com/eProsima/Micro-XRCE-DDS) and [Micro-ROS](https://micro.ros.org/)
```
Comment (fujitatomoya) [inline sig-robotics/charter.md:23]: adding Micro-ROS aligned with XRCE-DDS.

**Comment by fujitatomoya** [comment]
@JoeyHwong-gk could you take a look when you have time?

**Comment by JoeyHwong-gk** [comment]
> @JoeyHwong-gk could you take a look when you have time?

Thank you for the pull request! Your contributions are much appreciated. 
I've reviewed the changes, and looks good to me.

/cc @Poorunga @fisherxu @dingyin @crystaldust

**Review by kevin-wangzefeng — COMMENTED** [review]
/approve
Thanks

**Pull request merged**